package.json

```diff
@@ -48,12 +50,12 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^3.0.0",
     "location-bar": "^3.0.1",
-    "lodash": "^3.10.1",
+    "lodash": "^4.17.12",
     "markdown-toc": "^0.11.7",
     "marked": "^0.3.5",
     "mini-css-extract-plugin": "^0.4.1",
     "minimist": "^1.1.1",
-    "moment": "2.25.3",
+    "moment": "^2.25.3",
```

Comment:
We need to use the exact version of moment here.

Reply:
done